test/example/TestFuser.test_lstm_cuda-backward.expect

```diff
@@ -0,0 +1,111 @@
+graph(%0 : Float(*, *),
+      %1 : Float(*, *),
+      %2 : UndefinedTensor,
```

Comment:
this might not be an up-to-date graph now, UndefinedTensor is removed from the type system, so we might need to update it.